latex/README.md

```
@@ -0,0 +1,29 @@
+## LaTeX generation of the RFCs from the .md files
+
+
+For the VS Code, it's good to use plugin https://marketplace.visualstudio.com/items?itemName=James-Yu.latex-workshop.
+This extension already has settings saved in the repo for easy usage.
+
+
+### To run the generator:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)